File: php/02integer/integer7.php
<?php
$N = mt_rand(10, 99);
$a = intval($N / 10);
$b = $N % 10;
$sum = $a + $b;
$zarb = $a * $b;
echo "Адади дурақама: ".$N;
echo "<br>Қисми даҳӣ: ".$a;
echo "<br>Қисми воҳидӣ: ".$b;
echo "<br>Суммаи рақамҳо: ".$sum;
echo "<br>ҲосилиЗарби рақамҳо: ".$zarb;
?>

File: cpp/02integer/Integer7.cpp
#include <iostream>
using namespace std;

int main()
{
	//Task("Integer7");
	int number;
	cin >> number;
	int a = number / 10;
	int b = number % 10;
	int sum = a+b;
	int zarb = a*b;
	cout << sum << zarb;
	
	return 0;
}
